avplayback.js

```diff
@@ -78,27 +78,27 @@ instance.prototype.actions = function(system) {
 
 	self.system.emit('instance_actions', self.id, {
 		'load': {
-			label: 'Load Clip (nr)',
+			label: 'Load Clip (nr) e.g. 01',
 			options: [
 				{
 					 type: 'textinput',
 					 label: 'Clip Nr.',
 					 id: 'clip',
-					 default: 1,
-					 regex: self.REGEX_NUMBER
+					 default: "01",
```

Comment:
I think the user of companion should not be given the responsibility to prepend the 0 in the numbers. So this section should not be changed.